commit 29df6a2f2036c4d25b3fa6d8bc6e8e35593ec15d
Author: Tim Gardner <tim.gardner@canonical.com>
Date:   Fri Sep 23 09:05:03 2016 -0600

    Revert "UBUNTU: SAUCE: Clear Linux: bootstats: add printk's to measure boot time in more detail"
    
    BugLink: http://bugs.launchpad.net/bugs/1626104
    
    This reverts commit 18b27273904ac9a79b3a84c0f8933b40203a365e.
    
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
    
     Conflicts:
            init/main.c

diff --git a/arch/x86/kernel/alternative.c b/arch/x86/kernel/alternative.c
index da6b82f..5cb272a 100644
--- a/arch/x86/kernel/alternative.c
+++ b/arch/x86/kernel/alternative.c
@@ -626,9 +626,7 @@ void __init alternative_instructions(void)
 	 * patching.
 	 */
 
-	printk("Applying alternatives\n");
 	apply_alternatives(__alt_instructions, __alt_instructions_end);
-	printk("Applying alternatives done\n");
 
 #ifdef CONFIG_SMP
 	/* Patch to UP if other cpus not imminent. */
@@ -639,8 +637,6 @@ void __init alternative_instructions(void)
 					    _text, _etext);
 	}
 
-	printk("Applying alternatives smp done\n");
-
 	if (!uniproc_patched || num_possible_cpus() == 1)
 		free_init_pages("SMP alternatives",
 				(unsigned long)__smp_locks,
diff --git a/drivers/base/firmware_class.c b/drivers/base/firmware_class.c
index b987036..22d1760 100644
--- a/drivers/base/firmware_class.c
+++ b/drivers/base/firmware_class.c
@@ -1223,7 +1223,6 @@ request_firmware(const struct firmware **firmware_p, const char *name,
 {
 	int ret;
 
-	printk("request_firmware: %s\n", name);
 	/* Need to pin this module until return */
 	__module_get(THIS_MODULE);
 	ret = _request_firmware(firmware_p, name, device, NULL, 0,
diff --git a/kernel/kmod.c b/kernel/kmod.c
index d98c70f..0277d12 100644
--- a/kernel/kmod.c
+++ b/kernel/kmod.c
@@ -76,8 +76,6 @@ static int call_modprobe(char *module_name, int wait)
 		NULL
 	};
 
-	printk("call_modprobe: %s   %i \n", module_name, wait);
-
 	char **argv = kmalloc(sizeof(char *[5]), GFP_KERNEL);
 	if (!argv)
 		goto out;
